app/mailers/invite_people_mailer.rb

```diff
@@ -13,7 +13,7 @@ def to_start_group(invitation:,
 
   def to_join_group(invitation:, locale:, subject_key: "email.to_join_group.subject")
     @invitation = invitation
-    @message_body = invitation.message
+    @message_body = invitation.message unless invitation.message.blank?
```

Comment:
I don't get this; why have a check for blank here? Isn't it okay if it's blank since we do

```
- if @message_body
```

later?